src/types.ts

```diff
@@ -11,7 +11,7 @@ export const WebpubMimeType = 'application/webpub';
  * THE FOLLOWING TWO TYPES ARE MADE UP STUBS
  */
 // a webpub pdf collection
-export const WebpubPdfConformsTo = 'stub/webpub+pdf';
+export const WebpubPdfConformsTo = 'application/webpub+pdf';
```

Comment:
I would leave this as is right now to be clear it's just a stub... eventually it will be replaced with whatever we decide with mike. but not a big deal either